### PR TITLE
Isbn website

### PR DESCRIPTION
**Pull request by Priyanshii677**

### CODE REVIEW

File: googlebookapi.yaml [hunk 1]
```diff
@@ -43,4 +52,7 @@ spec:
   - name: tcp
     port: 9999
     targetPort: 9999
+  - name: tcp
```
Comment (mgdevstack) [inline googlebookapi.yaml]:
```suggestion
  - name: ui-tcp
```

File: googlebookapi.yaml [hunk 2]
```diff
@@ -43,4 +52,7 @@ spec:
   - name: tcp
```
Comment (mgdevstack) [inline googlebookapi.yaml]:
```suggestion
  - name: bookapi-tcp
```

File: ui/templates/hello.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        <input
+          type="text"
+          placeholder="Search.."
+          class="searchbox"
+          id="myForm"
+        />
+        <button onclick="myFunction()" class="button" id="myBtn">Go</button>
+      </div>
+      <div class="example">
+        <input
```
Comment (mgdevstack) [inline ui/templates/hello.html]: Just ISBN numbers are mentioned. Add `Examples:` xxxxxxxx , xxxxxxxxxx etc

File: ui/templates/hello.html [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+          onclick="exampleFunction2()"
+        />
+      </div>
+    </div>
+
+    <script>
+      function myFunction() {
+        var y = document.getElementById("myForm").value;
+        console.log(y);
+        window.location.href = " http://roost-worker:30045/books/" + y;
```
Comment (mgdevstack) [inline ui/templates/hello.html]: Host of the endpoint should be read from ENV["API_ENDPOINT"]

Comment (mgdevstack) [reply ui/templates/hello.html]: JS should make and API call to get book details. And JSON response returned from the API should be displayed in the same UI page at the bottom as Book Detail. (No redirection to http://$HOST/books/)

**Review by mgdevstack — CHANGES_REQUESTED** [review]

File: ui/templates/hello.html (hunk 4 above)
Comment (mgdevstack) [reply ui/templates/hello.html]: ENV["API_ENDPOINT"] can be configured in it's container section of googlebookapi.yaml

**Comment by harishagrawal** [comment]
ui.exe is checked in .. 
Shouldn't this be getting created within the docker container

**Review by harishagrawal — APPROVED** [review]

**Pull request merged**